docs/user/alerting/rule-types.asciidoc

```diff
@@ -20,40 +20,50 @@ see {subscriptions}[the subscription page].
 [cols="2*<"]
```

Comment:
There is a section above that says `{kib} provides two types of rules: rules specific to the Elastic Stack and rules specific to a domain.`. Should that be reworded as well? Should we remove the references to "two types of rules"? Maybe `{kib} provides rules built into the Elastic Stack and rules registered by one of the {kib} apps.`